lms/djangoapps/courseware/courses.py

```diff
@@ -441,6 +441,9 @@ def get_courses(user, org=None, filter_=None):
         settings.COURSE_CATALOG_VISIBILITY_PERMISSION
     )
 
+    print courses
+    for c in courses:
+        print '{}: {}'.format(c, has_access(user, permission_name, c))
```

Comment:
Is this leftover debug stuff, or here for a reason?

Reply:
This is from a bogus commit due to all my rebasing. I'm going to remove it.